fpgaperf.py

```diff
@@ -382,6 +403,7 @@ def main():
                 print('{}: error: {}'.format(sys.argv[0], e))
             sys.exit(1)
 
+        logger.debug("Continuing")
```

Comment:
Probably this is not useful as it does not give information on what is happening

Reply:
Good point. `Preparing Project` would probably be a better descriptor for what is happening there. I'll change that.

Reply:
Hmm maybe you could actually skip that entirely as I already see there is something similar (`Starting Project`). Might be confusing

Reply:
My thought process was that first, it prepares the project (with get_project(project) and specifying the board info and toolchain info that it would need) and then (after `Getting Constraints`) it starts the project with t.project() and runs the project with t.run(). Would you still like me to get rid of the Preparing Project? Another option could be to remove the `Starting Project` and move `Running Project` to be before t.project(). What do you think?

Reply:
I believe the second option is all right (replace starting project with running project)

Reply:
One more thing is that you probably should run `make format` to have the autoformatter clean the python changes and conform them to the standard used for this project